run.py

```diff
@@ -1,24 +1,54 @@
 #!/home/mr124/PyEnv/E-Memory/bin/python3
 
-import sys
 from FlaskSite import createApp
+import subprocess
+import sys
+
 
-def getRunConfiguration():
+def get_run_configuration():
     """Determine the run configuration based on command-line arguments."""
     return len(sys.argv) == 2
 
+
+def generate_requirements():
```

Comment:
well-structured and commented code 👍
just that IOError could be handled to ensure file operations. .
Great job overall!

Reply:
you mean change `subprocess.CalledProcessError` to general exception or add another except for IOError?